e2e/firstTest.spec.js

```diff
@@ -0,0 +1,19 @@
+describe('Example', () => {
```

Comment:
Is that file necessary? Looks like a dummy file that won't be run, since we run tests from `__e2e__`, but not `e2e`

Reply:
Yes I have removed the e2e folder completely. It was not being used.